packages/@okta/vuepress-site/docs/guides/express-configuration/main/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Express Configuration overview
+excerpt: An overview of Express Configuration.
```

Comment:
Can you use the excerpt to describe what is covered in the overview. Avoid just repeating the title here.

```suggestion
excerpt: An overview of Express Configuration.
```